roles/ood_user_reg/tasks/main.yml

```diff
@@ -66,3 +66,29 @@
 - name: Build the updated Apache config
   command: /opt/ood/ood-portal-generator/sbin/update_ood_portal
   ignore_errors: yes
+
+- name: Create RegUser - {{ RegUser_app_user }} on OHPC
+  shell: ssh ohpc 'sudo id -u {{ RegUser_app_user }} &>/dev/null || sudo useradd -c "{{ RegUser_app_user_full_name }}" {{ RegUser_app_user }}'
+  become_user: vagrant
+
+- name : delay
+  command: sleep 2
+
+- name: Change passwd RegUser - {{ RegUser_app_user }} on OHPC
+  shell: ssh ohpc 'sudo echo "{{ RegUser_app_user_passwd }}" | sudo passwd --stdin "{{ RegUser_app_user }}"'
+  become_user: vagrant
+
+- name: Create Cluster keys for RegUser - {{ RegUser_app_user }} on OHPC
+  shell: ssh ohpc 'sudo -s su - reggie -c "sh /usr/bin/cluster-env"'
+  become_user: vagrant
+
+- name: Get UID of RegUser - {{ RegUser_app_user }} from OHPC
```

Comment:
Do this on ood via inspection of uid, gid owner of /home/reggie.  Then can use those uid, gid in the user module create request within ansible task on ood role.